internal/poller/gotimestatus/gotimestatus.go

```diff
@@ -0,0 +1,188 @@
+package gotimestatus
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"sort"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+// timeNow returns the current time
+// it is unexported and overridable to facilitate testing
+var timeNow = time.Now
```

Comment:
I'd rather inject the `time.Now` as a parameter to the `New()` constructor of `GoTimeStatus` and have that as a struct member than a global variable.

Reply:
Its not really an option that should be tweaked from the public API; it's only valuable for testing.

If the primary gripe is using a global var instead of something more local to the struct, i've moved it into the struct as a member value.